### ISSUE TITLE
Config File

### ISSUE THREAD

**Issue by travankor** [cylgom/ly#51]
Instead of having to recompile everytime you want to change a setting, ly could have a config file that you can change settings without recompiling. 

This would be useful when ly has more options, which seems likely if this project keeps being developed ;)

**Comment by slashome** [comment]
I agree with this, I'm migrating more and more to CLI / ncurses apps but I would be more confortable with a configuration file so I can keep update easily app from pacman :)

**Comment by cylgom** [comment]
You will be pleased to learn I'm currently re-writing Ly, and already implemented a config file system :) Since Ly was my first C project to be released, this new version will be significantly better.

**Comment by slashome** [comment]
> You will be pleased to learn I'm currently re-writing Ly, and already implemented a config file system :) Since Ly was my first C project to be released, this new version will be significantly better.

That an amazing news ! Can't wait to test it !

**Comment by vontamar** [comment]
Can't wait!

**Comment by cylgom** [comment]
Some news.

It's becoming better and better :) Get ready for:
- a clean codebase (development restarted from scratch)
- termbox (replaces ncurses for the greater good)
- tcc compilation (to support the only sane compiler out there)
- configurability (change everything with a text file)
- translations (only english and french for now)
- truecolor support (which is usless... but cool)
- background animations support (I have no valid reason for this I'm just insane)
- auto-refresh (byebye log pollution)

Save file, auto-tty switch and f-keys support is still to be done.
I also want to add keyboard map settings, and caps lock and numlock info.

I need some time to work on the packages and the readme as well. Stay tuned :)
